spec/factories_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 RSpec.describe 'FactoryBot' do
   FactoryBot.factories.each do |factory|
+    next if factory.name == :action_text_rich_text
```

Comment:
@JoshDevHub 
I think unless we can somehow have a fake ActiveRecord model with a fake database table, it’s best to skip linting the ActionText::RichText factory.